From timothy.sipples@us.ibm.com Fri Jun 16 04:36:01 2006
From: Timothy Sipples <Timothy.Sipples@us.ibm.com>
To: greg@kroah.com
Subject: airprime.c: add Kyocera Wireless KPC650/Passport support
Message-ID: <OF9CDBE0E1.1FFB454D-ON4925718F.003F5473-4925718F.003F9499@us.ibm.com>
Date: Fri, 16 Jun 2006 20:42:59 +0900

From: Timothy Sipples <Timothy.Sipples@us.ibm.com>

Adds vendor & product IDs to drivers/usb/serial/airprime.c for Kyocera 
Wireless KPC650/Passport EV-DO/1xRTT PC Cards.

Signed-off-by: Timothy Sipples <Timothy.Sipples@us.ibm.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/airprime.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/usb/serial/airprime.c
+++ gregkh-2.6/drivers/usb/serial/airprime.c
@@ -16,6 +16,7 @@
 #include "usb-serial.h"
 
 static struct usb_device_id id_table [] = {
+	{ USB_DEVICE(0x0c88, 0x17da) }, /* Kyocera Wireless KPC650/Passport */
 	{ USB_DEVICE(0xf3d, 0x0112) },  /* AirPrime CDMA Wireless PC Card */
 	{ USB_DEVICE(0x1410, 0x1110) }, /* Novatel Wireless Merlin CDMA */
 	{ USB_DEVICE(0x1199, 0x0112) }, /* Sierra Wireless Aircard 580 */
